### PR TITLE
Update: assigned new decorator for admin access

### PR DESCRIPTION
**Pull request by VishalMinj**
## Changes:
- Added conditional rendering in base.html

## Updates:
- Assigned staff_member_required to add_movie, this will automatically pop up the django admin login pannel and also redirect to the add_movie view

### CODE REVIEW

**Comment by VishalMinj** [comment]
How to update the pr link at ContiHUB website?

**Comment by eleensmathew** [comment]
> How to update the pr link at ContiHUB website?

No need to update it. Can you add the decorator for add_show view as well

File: user/views.py [hunk 1]
```diff
@@ -41,6 +42,7 @@ def get(self, request):
 
 
 # it will fetch data from api if only title field is  provided 
+@staff_member_required
```
Comment (eleensmathew) [inline user/views.py:46]: Add decorator for add_show as well

Comment (VishalMinj) [reply user/views.py:46]: okay

Comment (VishalMinj) [reply user/views.py:46]: You mean the AddshowView right?

Comment (eleensmathew) [reply user/views.py:46]: Yup

**Pull request merged**